nav2_bringup/params/nav2_multirobot_params.yaml

```diff
@@ -0,0 +1,282 @@
+amcl:
```

Comment:
This is not a specific suggestion but a question: 

Can you think of any other names for the config file / launch file to help distinguish it from the other multirobot launch file so that its more clear to users what each will be doing on a quick glance? I alluded to this earlier as well - we could rename the other multirobot launch file as well.

Reply:
Well, yeah, like you mentiond. I think `multi-robot` keyword it's not definetly proper in this launch/config file.
Because 'bringup_multirobot_launch' file does not bring up multiple number of robot literally. Actual behavior is just appling namespace.

So my opinion is ... how about 'bringup_with_ns_launch.py' and 'nav2_namespaced_params.yaml'? 
Do you have any good naming?

Reply:
Why not literally make it bring up multiple robots with the loop like the other multirobot launch file does? That seems to be the intent from the PRs name and the original changes? That would be my preference. You could always make the number of robots = 1 by default :-)